src/python/serenity_types/pricing/derivatives/rates/yield_curve.py

```diff
@@ -149,7 +149,7 @@ class YieldCurveDefinition(BaseModel):
 
     display_name: str
     """
-    Human-readable descrition of this curve, e.g. OIS (RAW) or OIS (Interpolated, FLAT_FWD)
+    Human-readable description of this curve, e.g. OIS (RAW) or OIS (Interpolated, FLAT_FWD)
```

Comment:
Actually, with most recent type you don't need to include RAW / INTERPOLATED, so it should just be "OIS (FLAT_FWD)" as the example.